claudia-lab/gulpfile.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
```

Comment:
@loomnugget Good gulpfile - remember to add, remove, or update specific pieces (or tasks) as you need them.  This file is meant to help you with your specific workflow and we are totally fine with you modifying this to fit your workflow needs.